frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-query-result/experiment-query-result.component.ts

```diff
@@ -29,20 +44,27 @@ export class ExperimentQueryResultComponent implements OnInit, OnDestroy {
   };
 
   queryResults = {};
-  queryFactorResults1 = {};
-  queryFactorResults2 = {};
-  interactionEffectQueryFactorResults1 = {};
-  interactionEffectQueryFactorResults2 = {};
+  queryFactorResults = [];
+  interactionEffectQueryFactorResults = [];
   queryResultsSub: Subscription;
   isQueryExecuting$ = this.analysisService.isQueryExecuting$;
-  factors = [];
+  factors: string[] = [];
+  queries: string[] = [];
   displayedColumns: string[] = [];
   factorialData = {};
   experimentType: string = null;
   data: { name: string; series: { name: string; value: number }[]; dot: boolean }[];
   meanData2: { name: string; value: number }[];
   meanData1: { name: string; value: number }[];
   maxLevelCount = 0;
+  factorColumnDefs: FactorColumnDef[] = [];
+  queryColumnDefs: QueryColumnDef[] = [];
+  dataSource: RowData[] = [];
+
+  // dataSource = [
```

Comment:
this looks like some commented old debug code, can it be removed?

Reply:
Oh ...missed that..thanks..Also updated few things for graphs UI polishing in the next commit.